src/BizHawk.Client.EmuHawk/tools/SNES/SNESGraphicsDebugger.cs

```diff
@@ -1263,15 +1268,16 @@ private void MessageTimer_Tick(object sender, EventArgs e)
 		private void comboPalette_SelectedIndexChanged(object sender, EventArgs e)
 		{
 			if (suppression) return;
-			var pal = (SnesColors.ColorType)comboPalette.SelectedValue;
+			var pal = comboPalette.SelectedValue.ToString();
 			Console.WriteLine("set {0}", pal);
-			var s = Emulator.GetSettings();
-			s.Palette = pal.ToString();
-			currentSnesCore?.PutSettings(s);
+			currentSnesCore?.SetPalette((SnesColors.ColorType)comboPalette.SelectedIndex);
```

Comment:
`pal` is now unused, what did you intend?

Reply:
Well, it's now only used in the `Console.WriteLine` call, which I just kept from before.